packages/start-plugin-core/tests/importProtection/transform.test.ts

```
@@ -0,0 +1,197 @@
+import { describe, expect, test } from 'vitest'
+import { rewriteDeniedImports } from '../../src/import-protection-plugin/rewriteDeniedImports'
+import { RESOLVED_MOCK_MODULE_ID } from '../../src/import-protection-plugin/virtualModules'
```

ESLint

[error] 'RESOLVED_MOCK_MODULE_ID' is defined but never used.

(unused-imports/no-unused-imports)